downloads/factories.py

```diff
@@ -0,0 +1,112 @@
+from urllib.parse import urljoin
+
+import factory
+import requests
+
+from users.factories import UserFactory
+
+from .models import OS, Release, ReleaseFile
+
+
+class OSFactory(factory.DjangoModelFactory):
+
+    class Meta:
+        model = OS
+        django_get_or_create = ('slug',)
+
+    creator = factory.SubFactory(UserFactory)
+
+
+class ReleaseFactory(factory.DjangoModelFactory):
+
+    class Meta:
+        model = Release
+        django_get_or_create = ('slug',)
+
+    creator = factory.SubFactory(UserFactory)
+    is_published = True
+
+
+class ReleaseFileFactory(factory.DjangoModelFactory):
+
+    class Meta:
+        model = ReleaseFile
+        django_get_or_create = ('slug',)
+
+    creator = factory.SubFactory(UserFactory)
+    release = factory.SubFactory(ReleaseFactory)
+    os = factory.SubFactory(OSFactory)
+
+
+class APISession(requests.Session):
+    base_url = 'https://www.python.org/api/v1/'
```

Comment:
The new API will be deployed to production on this Friday. I would like to use v2 here. If you have some time before Friday, you can try it out at https://staging.python.org/api/v2/